routemodel/solar_rad/solcast-api-test.py

```diff
@@ -0,0 +1,26 @@
+import json
+import requests
+
+api_key = 'na'
+base_url = 'https://api.solcast.com.au/'
+
+headers = {'Content-Type': 'application/json',
+           'Authorization': 'Bearer {0}'.format(api_key)}
+
+def get_radiation_data(latitude, longitude):
+    #creating dictionary of parameters (not sure if I should be creating this here or not)
+    params = {
+        'latitude': latitude,
+        'longitude': longitude
+    }
+
+    api_url = base_url + '/world_radiation/forecasts'
+    response = requests.get(api_url, headers=headers, params=params)
+    if response.status_code == 200:
+        return json.loads(response.content.decode('utf-8'))
+    else:
+        return None
+
+#driver code
```

Comment:
Try wrapping this in an if statement (`if __name__=="__main__":`), otherwise CI will flag it